_includes/open_graph.html

```diff
@@ -0,0 +1,25 @@
+<meta property="og:url" content="{{ site.url }}{{ page.url }}" />
+<meta property="og:type" content="article" />
+<meta property="og:title" content="{{ page.title | default: site.title }}" />
+{% if page.summary %}
+  {% assign description = page.summary %}
+{% elsif page.questions %}
+  {% assign description = page.questions | join: " " %}
+{% elsif page.keypoints %}
+  {% assign description = page.keypoints | join: " " %}
+{% elsif include.description %}
+  {% assign description = include.description %}
+{% else %}
+  {% assign description = "Learning guide for instructors and learners, with the opportunity for live coding examples." %}
+{% endif %}
+
+{% assign description = description | strip_html %}
+{% if description.size > 200 %}
+  {% assign description = description | truncate: 200 %}
+  {% assign description = description | append: "..." %}
+{% endif %}
+<meta property="og:description" content="{{ description }}" />
+<meta property="og:image" content="{{ site.url }}/assets/favicons/cp/mstile-310x310.png" />
```

Comment:
This allows us to use the icons for the specific curriculum instead of only Carpentries. 

```suggestion
{% assign favicon_url = relative_root_path | append: '/assets/favicons/' | append: site.carpentry  %}
<meta property="og:image" content="{{ favicon_url }}/mstile-310x310.png" />
```